platform/src/main/resources/config/anchor-config-default-values.yaml

```diff
@@ -0,0 +1,487 @@
+    ##    If type is JWT_TOKEN - the secret key of JWT encryption
+    ##    If type is API_KEY: -
+    ##        PlatformAPI requests (`Anchor->Platform`) will contain an
+    ##        authentication header whose token was built using this secret. The Platform Server will
+    ##        use this secret to decode the incoming token to verify it came from the Anchor.
+    ##        also store this same secret and use it to decode the incoming token to verify it
+    ##        came from the Platform.
+    #
+    type: NONE
+
+    ## @param: expiration_milliseconds
+    ## @type: int
+    ## Expiration time, in milliseconds, that will be used to build and validate the JWT tokens
+    #
+    expiration_milliseconds: 30000
+
+payment_observer:
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable a circle payment observer
+  #
+  enabled: false
+
+  ## @param: circle_url
+  ## @type: string
+  ## Circle api endpoint.
+  #
+  circle_url: https://api-sandbox.circle.com
+
+
+  ## @param: tracked_wallet
+  ## @type: string
+  ## The id of the wallet to track. 'all' indicates that events for all wallets are tracked.
+  #
+  tracked_wallet: all
+
+
+## @param: languages
+## @supported_values: en
+## The supported languages defined by RFC4646 (https://datatracker.ietf.org/doc/html/rfc4646)
+#
+languages: en
+
+logging:
+  ## @param: level
+  ## @type: string
+  ## The root logger logging level
+  level: INFO
+
+  ## @param: level
+  ## @type: string
+  ## The "org.stellar" logger logging level.
+  stellar_level: INFO
+
+######################
+## SEP configuration
+######################
+sep1:
+
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-1 for this instance of AP server.
+  #
+  enabled: false
+
+  toml:
+    ## @param: type
+    ## @type: string
+    ## @supported_values:
+    ##     `string`: value contains the content of the stellar toml file
+    ##     `file`: value contains the path to the stellar toml file
+    ##     `url`: value contains the url to the content of the stellar toml file
+    type: string
+
+    ## @param: value
+    ## @type: string
+    #
+    value: |
+      
+
+sep10:
+
+  ## @param: enabled
+  ## @type: bool
+  ## Whether to enable SEP-10 for this instance of AP server.
+  ## If true, @required_secrets: SEP10_SIGNING_SEED
+  #
+  enabled: false
+
+  ## If `enabled` is set to True, the following two secrets must be set via the environment
+  ## variables.
+  ## `secret.sep10_signing_seed`: The private key for signing challenge transactions.
+  ## `secret.sep10_jwt_secret`: The JWT encryption key.
+  #
+
+  ## @param: home_domain
+  ## @type: string
+  ## The `home_domain` property of SEP-10. https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0010.md#request
+  #
+  home_domain: localhost:8080
+
+  ## @param: client_attribution_required
+  ## @type: bool
+  ## Set if the client attribution is required
+  #
+  client_attribution_required: false
```

Comment:
There will be another round of documentation review led by @JakeUrban . In order to make it consistent, I would leave this to Jake at this moment.

Reply:
"Client attribution" is an unofficial name that I've been using since we created the capability in SEP-10, but unfortunately there is no mention of that term in the SEP-10 standard.

That being said, I think this is the right name for the config option, and Polaris uses the same terminology. When I do this 2nd round of documentation I'll make sure to add that term to the official spec too.

Reply:
Thanks, @JakeUrban